articles/react-native-firebase-facebook-authentication/index.md

```diff
@@ -0,0 +1,451 @@
+
+Make sure you're following the React Native CLI Quickstart, not the Expo CLI Quickstart.
+
+![Env Setup](env_setup.png)
+
+### Cloning the starter code
+
+Clone the starter code from this [GitHub repository](https://github.com/zolomohan/rn-firebase-phone-auth-starter).
+
+You can check out the final code from this [GitHub Repository](https://github.com/zolomohan/rn-firebase-phone-auth).
+
+This will be the folder structure of the application.
+
+![Folder Structure](folder_structure.png)
+
+I've set up 2 screens in the `screens/` directory:
+
+- _Authentication.js_: Screen with a Facebook Sign-in button to initiate the sign-in process.
+
+- _Authenticated.js_: Screen that the user can see only if he is logged in.
+
+![Screens](screens.jpg)
+
+### Setting up the Firebase project
+
+Head to the [Firebase console](https://console.firebase.google.com/u/0/), sign in to your account and Create a new project.
+
+![Create New Project](firebase_new.png)
+
+You'll be presented with the dashboard once you create the new project.
+
+![New Dashboard](new_dashboard.png)
+
+Now, click on the Android icon to add an android app to the Firebase project.
+
+![register_app](register_app.png)
+
+You'll need the package name of the application to register your application. You'll find the package name in the `AndroidManifest.xml` which is located in `android/app/src/main/`.
+
+![Package Name](package_name.png)
+
+In the next step, you should download the `google-services.json` file and place it in the `android/app` directory.
+
+This file contains configurations that'll enable your application to access firebase services.
+
+![Download Google Services JSON](download_services.json.png)
+
+In the next step, It'll ask you to add configurations to the `build.gradle` files.
+
+First, add the `google-services` plugin as a dependency inside of your `android/build.gradle` file:
+
+```gradle
+buildscript {
+  dependencies {
+    // ... other dependencies
+    classpath 'com.google.gms:google-services:4.3.3'
+  }
+}
+```
+
+Then, execute the plugin by adding the following to your `android/app/build.gradle` file:
+
+```gradle
+apply plugin: 'com.android.application'
+apply plugin: 'com.google.gms.google-services'
+```
+
+You need to perform some additional steps to configure `Firebase` for `iOS`. Follow [this documentation](https://rnfirebase.io/#3-ios-setup) to set it up.
+
+Finally, let's install the `@react-native-firebase/app` package in our app to complete the set up for Firebase.
+
+```bash
+npm install @react-native-firebase/app
+```
+
+### Setting up Facebook App
+
+Head to [developers.facebook.com](https://developers.facebook.com/) and create a developer account.
+
+Go to the [Apps](https://developers.facebook.com/apps) section and create a new app.
+
+![Create FB App](create_app.png)
+
+Once you create an application, you will be greeted with the App Dashboard. Add the Facebook Login product to your app from the dashboard.
+
+![App Dashboard](fb_dashboard.png)
```

Comment:
Please change the image paths to the full paths

`/engineering-education/react-native-firebase-facebook-authentication/imagename`

Reply:
updated them.